wallet/test/Test/Pos/Util/MnemonicSpec.hs

```diff
@@ -65,31 +79,38 @@ spec = do
         it "Can generate 128 bits entropy" $
             (length . entropyToByteString <$> genEntropy @128) `shouldReturn` 16
 
-        it "Mnemonic to JSON" $ forM_ testVectors $ \(bytes, _, mnemonic, _, _) ->
+        it "Mnemonic to JSON" $ forM_ testVectors $ \(bytes, _, _, mnemonic, _, _) ->
             Aeson.encode mnemonic `shouldBe` bytes
 
-        it "Mnemonic from JSON" $ forM_ testVectors $ \(bytes, _, mnemonic, _, _) ->
+        it "Mnemonic from JSON" $ forM_ testVectors $ \(bytes, _, _, mnemonic, _, _) ->
             Aeson.decode bytes `shouldBe` pure mnemonic
 
-        it "CBackupPhrase to JSON" $ forM_ testVectors $ \(bytes, _, mnemonic, _, _) ->
+        it "CBackupPhrase to JSON" $ forM_ testVectors $ \(bytes, _, _, mnemonic, _, _) ->
             Aeson.encode (CBackupPhrase mnemonic) `shouldBe` jsonV0Compat bytes
 
-        it "CBackupPhrase from JSON" $ forM_ testVectors $ \(bytes, _, mnemonic, _, _) ->
+        it "CBackupPhrase from JSON" $ forM_ testVectors $ \(bytes, _, _, mnemonic, _, _) ->
```

Comment:
I am not sure I understand this properly - but shouldn't this be called "CBackupPhrase from legacy JSON" as we are using `jsonV0Compat` bellow?

This is new to me so I might have wrong understanding

Reply:
I think the naming here is in contrast to the next one, which *is* called "CBackupPhrase from legacy JSON". 

The difference being `bytes` vs `legacyBytes`. Might exist better ways.